src/main/java/com/sandvoxel/generitech/common/blocks/Blocks.java

```diff
@@ -59,7 +59,7 @@
     BLOCK_PULVERIZER(BlockPulverizer.class, ItemPulverizer.class),
 
     BLOCK_MACHINEMATRICS(BlockMachineMatrics.class, ItemMachineMatrics.class),
-    BLOCK_CABLES(BlockCables.class, ItemCable.class);
+    BLOCK_GEN(BlockGen.class, ItemCable.class);
 
     private final Class<? extends BlockBase> blockClass;
     private final Class<? extends ItemBlock> itemBlockClass;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused private "Blocks" constructor. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar.aadev.xyz/coding_rules#rule_key=squid%3AUnusedPrivateMethod)